### DISCUSSION TITLE
Are all repo contents prevented from being crawled by search engines?

### DISCUSSION THREAD

**Discussion by zhiliangxu** [community/community#22746] [New to GitHub]
<p>Hello GitHub experts!</p>
<p>Recently I found that the <a href="https://github.com/robots.txt" rel="nofollow noopener">https://github.com/robots.txt</a> has prevented all repo contents from being crawled by search engines. Here is the current content of the robots.txt.</p>
<pre><code class="lang-auto"># If you would like to crawl GitHub contact us at support@github.com.
# We also provide an extensive API: https://developer.github.com/

User-agent: baidu
crawl-delay: 1

User-agent: *

Disallow: */pulse
Disallow: */tree/
Disallow: */blob/
Disallow: */wiki/
Disallow: /gist/
Disallow: */forks
Disallow: */stars
Disallow: */download
Disallow: */revisions
Disallow: */issues/new
Disallow: */issues/search
Disallow: */commits/
Disallow: */commits/*?author
Disallow: */commits/*?path
Disallow: */branches
Disallow: */tags
Disallow: */contributors
Disallow: */comments
Disallow: */stargazers
Disallow: */archive/
Disallow: */blame/
Disallow: */watchers
Disallow: */network
Disallow: */graphs
Disallow: */raw/
Disallow: */compare/
Disallow: */cache/
Disallow: /.git/
Disallow: */.git/
Disallow: /*.git$
Disallow: /search/advanced
Disallow: /search
Disallow: */search
Disallow: /*q=
Disallow: /*.atom

Disallow: /ekansa/Open-Context-Data
Disallow: /ekansa/opencontext-*
Disallow: */tarball/
Disallow: */zipball/

Disallow: /account-login
Disallow: /Explodingstuff/
</code></pre>
<p>I believe that by <code>Disallow: */blob/</code>, none of repo contents can be crawled.<br>
I remember that previously GitHub allows master branch to be crawled. Could you share what’s the intention of this change? What should I do if I want my repo contents to be indexed by Google?</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi <a class="mention" href="/zhiliangxu">@zhiliangxu</a>! :wave: Welcome to the Community!</p>
<p>I’ve been checking with our SEO team about this - it seems that our robots.txt was updated in May to block all crawling, but we are in the process of reverting this change.</p>
<p>We expect repositories to start getting indexed again in the next few weeks!</p>

**Reply by willbeason** [reply]
Any update on this? The support link in robots.txt is broken now as the file hasn't been updated: https://support.github.com?tags=dotcom-robots

**Comment by pablo-mayrgundter** [comment]
Hi, any update on this?
